### PR TITLE
Exposes codebuilds 'cache_bucket_suffix_enabled' variable.

### PR DESCRIPTION
**Pull request by lezavala**
## what
* Exposes the code build variable `var.cache_bucket_suffix_enabled`

## why
* Allowing me to set to `false` prevents the random string generation to the S3 bucket suffix. This resolves an error I am getting:
```
Error: cache location is required when cache type is "S3"

  on .terraform/modules/test_build/main.tf line 206, in resource "aws_codebuild_project" "default":
 206: resource "aws_codebuild_project" "default" {
```

## references
* This seems to be related: https://github.com/hashicorp/terraform-provider-aws/issues/10195

### CODE REVIEW

File: variables.tf
Comment (Gowiem) [inline variables.tf:171]: Can you update the default to be `false` to keep the current behavior the same for folks who upgrade?

Comment (lezavala) [reply variables.tf:171]: I was under the impression `true` was the default behavior.
https://github.com/cloudposse/terraform-aws-codebuild/blob/3d9d73ac430e475486c75f7a6615c805f6837d09/variables.tf#L22

Should I proceed to set it as `false`?

Comment (Gowiem) [reply variables.tf:171]: Aha apologies -- you're correct 👍 

No need to update.

**Review by Gowiem — DISMISSED** [review]

**Comment by Gowiem** [comment]
@lezavala I'm working with the other contributors to figure out why this validate-codeowners check is not reporting. That is holding up merging this. Will hopefully merge next week.

**Comment by lezavala** [comment]
@Gowiem 
Any updates on this? Anything I can do on my end?

**Comment by SweetOps** [comment]
@lezavala , AFAIK there is no some workaround for this  issue at now, could you try create and push empty commit, example

```sh
git commit --allow-empty -m "Trigger GH actions"
git push
```

**Review by SweetOps — APPROVED** [review]

**Pull request merged**
